src/Exceptions/AnimationException.php

```diff
@@ -4,6 +4,6 @@
 
 namespace Intervention\Image\Exceptions;
 
-class AnimationException extends \RuntimeException
+class AnimationException extends RuntimeException
```

Comment:
This is better because it allows to catch all Exceptions with
```
catch (Intervention\Image\RuntimeException)
```
rather than
```
catch (\RuntimeException)
```
which is too generic.